packages/contractkit/src/wrappers/BaseWrapper.ts

```diff
@@ -30,7 +32,11 @@ export function toNumber(input: string) {
   return parseInt(input, 10)
 }
 
-export function parseNumber(input: string | number | BigNumber) {
+export function noopParser<A>(input: A): A {
```

Comment:
Naming nit: What do you think of calling this `identityParser` or something similar? No-op makes me think that maybe it wouldn't return anything. "Identity" fits in nicely with the functional paradigm we're using all over the place here.

Reply:
removed. I'm not using this